partcad/src/partcad/test/test.py

```
@@ -0,0 +1,57 @@
+        message = self._log_message_prepare(*args)
+        pc_logging.debug(f"Test: {shape.project_name}:{shape.name}: {self.name}{message}")
+
+    def failed(self, shape, *args) -> None:
+        """This methods works like logging.error() but prepends the message with the test name and the shape name."""
+        message = self._log_message_prepare(*args)
+        pc_logging.error(f"Test failed: {shape.project_name}:{shape.name}: {self.name}{message}")
+
+    def passed(self, shape, *args) -> None:
+        """This methods works like logging.error() but prepends the message with the test name and the shape name."""
+        message = self._log_message_prepare(*args)
+        pc_logging.info(f"Test passed: {shape.project_name}:{shape.name}: {self.name}{message}")
```

Ruff (0.8.2)

Logging statement uses f-string

(G004)